relayer-cli/src/conclude.rs

```diff
@@ -26,6 +26,18 @@
 //! Output::error(format!("{}", e)).exit();
 //! ```
 //!
+//! #### Note:
+//! The resulting output that this approach generates is determined by the 'error' annotation given
+//! to the error object `Kind::Query`. If this error object comprises any positional arguments,
+//! e.g. as achieved by `Query(String, String)`, then it is important to cover these arguments
+//! in the `error` annotation, for instance:
+//! ```compile_fail
```

Comment:
`ignore` is a more appropriate annotation here I believe, `compile_fail` should be reserved for snippets of code which must fail to compile for the documentation to be correct.

```suggestion
//! ```ignore
```

Reply:
This was initially `ignore` but [changed](https://github.com/informalsystems/ibc-rs/pull/518#discussion_r557326691) to `compile_fail`.

You're right, I didn't catch that subtlety! The documentation here fails to compile not as a way to assert its correctness, but because the module is not public..

https://github.com/informalsystems/ibc-rs/blob/80d222bbc6f28e0f0461393dec9f66a22e2ad975/relayer-cli/src/lib.rs#L22

I'll change to `ignore` which seems like a better option in this case. Thanks!

Reply:
4d78ed0